src/config/dynamic-theme-fixes.config

```diff
@@ -2863,6 +2863,42 @@ button
 
 ================================
 
+colors.dopely.top
+
+INVERT
+.footer
+.color-selector-icon
+
+CSS
+input.number-slider {
+    filter: invert(10%) saturate(0%) !important;
+}
+img[src*="svg"], .title {
+    filter: invert(50%) !important;
+}
+.accountDopely__letter,
+.editProfile__changeProfilePicture {
+    filter: saturate(0%) brightness(50%) !important;
+}
+div[class*="text"],
+a[class*="header"],
+.navbar-dropdown-item__logo,
+.navbar__listItem__button {
+    filter: saturate(0%) !important;
+}
+.palette-generator-container *,
+.toner-colors *,
+.toner-colors-container * {
```

Comment:
Their shouldn't be any inline styles happening on these. as it's already specified to ignore inline styles.